packages/shared-influx-pieces/src/register.ts

```diff
@@ -0,0 +1,7 @@
+import { ListenerStore, container } from '@skyra/http-framework';
+import { registerSchedule } from 'lib/utils/schedule.js';
+import { resolve } from 'path';
+
+container.stores.register(new ListenerStore().registerPath(resolve('listeners')));
```

Comment:
No... I meant `container.stores.get('listeners').registerPath(...)`.

Also, `path.resolve('listeners')` is basically `join(process.cwd(), 'listeners')`, it's not going to register on the directory's cwd.